godot-core/src/obj/gd.rs

```diff
@@ -549,13 +549,19 @@ where
     // https://github.com/godotengine/godot-cpp/issues/954
 
     unsafe fn from_arg_ptr(ptr: sys::GDExtensionTypePtr, call_type: PtrcallType) -> Self {
-        if T::Mem::pass_as_ref(call_type) {
-            let obj_ptr = interface_fn!(ref_get_object)(ptr as sys::GDExtensionRefPtr);
-            // ref_get_object increments the ref_count for us
-            Self::from_obj_sys_weak(obj_ptr)
+        let obj_ptr = if T::Mem::pass_as_ref(call_type) {
+            // ptr is `Ref<T>*`
```

Comment:
Maybe mention in a few words what `Ref<T>` is (for a reader not knowing Godot internals, it's not obvious).

It's also enough if you just add something like `(see GodotFfi for explanation)`, as it seems you already explain it elsewhere.